deepmd_cli/main.py

```diff
@@ -274,7 +274,11 @@ def main_parser() -> argparse.ArgumentParser:
         "-S", "--set-prefix", default="set", type=str, help="The set prefix"
     )
     parser_tst.add_argument(
-        "-n", "--numb-test", default=100, type=int, help="The number of data for test"
+        "-n",
+        "--numb-test",
+        default=100,
```

Comment:
is setting default to 0 a better option?

Reply:
I agree, although it might be a breaking change.